src/Flagbit/Zed/SprykerSmartInboxConnector/Communication/Controller/IndexController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Flagbit\Zed\SprykerSmartInboxConnector\Communication\Controller;
+
+use Spryker\Zed\Kernel\Communication\Controller\AbstractController;
+
+/**
+ * @method \Flagbit\Zed\SprykerSmartInboxConnector\Business\OneAndOneMailConnectorFacade getFacade()
+ * @method \Flagbit\Zed\SprykerSmartInboxConnector\Persistence\OneAndOneMailConnectorRepositoryInterface getRepository()
+ */
+class IndexController extends AbstractController
+{
+    /**
+     * @return array
+     */
+    public function indexAction()
+    {
+        return [
+            'foo' => 'hello worlkd',
```

Comment:
worlkd?

Reply:
:D in fact I guess we don't need this controller, so the whole class can be removed.